src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -127,7 +127,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         } elseif ($tag = $input->getOption('tag')) {
             $options = ['tag' => $tag, 'show_private' => $input->getOption('show-private')];
         } elseif ($name = $input->getArgument('name')) {
-            $name = $this->findProperServiceName($input, $errorIo, $object, $name);
+            $name = $this->findProperServiceName($input, $io, $object, $name);
```

Comment:
This one looks wrong to me, see my comment on the PR.

Reply:
This method ask the user to select a service. In 2 other commands we do it on stdout. So I don't see why it needs to be different here.

Reply:
This command handles multiple output formats including machine-readable ones. It uses stdout for contents that are meant to be redirected. Interactions are informational contents, nothing processable, hence they should go to stderr (e.g. to not pollute a xml result).

Reply:
Thanks for the explanation, I reverted. As an example, if you do `php bin/console debug:container webpack_encore --format=xml > foo.xml`, the choice selection is not shown if we output it on stdout.